test/e2e-tests.sh

```diff
@@ -65,47 +109,12 @@ kubectl delete buildtemplates --all
 
 # Run the tests
 
-local failed=0
+failed=0
+
 header "Running Go e2e tests"
 report_go_test -tags e2e ./test/e2e/... -count=1 || failed=1
 
-header "Running YAML e2e tests"
-ko apply -R -f test/ || failed=1
+run_yaml_tests || failed=1
```

Comment:
Should this be `&& failed=1` ?

If `run_yaml_tests` returns `0`, that'll be interpreted as `false` and therefore trip the `||` branch.

Reply:
No. In UNIX world, 0 means success.

Reply:
Right. I knew as I was typing it there was no way I had found a real bug. 😄